README.md

```diff
@@ -1,75 +1,42 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Capstone Project 1
 
-> One paragraph statement about the project.
+> In this project there are two main pages, home page and about page, designed in HTML, CSS and Javascript. The pages are designed to resemble this [template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). The featured speakers section in the home page is done with Javascript. 
 
-![screenshot](./app_screenshot.png)
+![Project screenshot](https://github.com/Fikerte-T/Capstone-Project-1/blob/home-about-pages/images/project-screenshot.png)
+[Project website] (https://fikerte-t.github.io/Capstone-Project-1/)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML
+- CSS
+- Javascript
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
+[git@github.com:Fikerte-T/Capstone-Project-1.git](git@github.com:Fikerte-T/Capstone-Project-1.git)
```

Comment:
- [ ] Your Pull Request does not include a video presentation showcasing your project. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The video should:
- contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- contain a demo of the project, showing the pages in different screen sizes.
- contain a highlight about a piece of the code the student finds interesting or is proud of making.